tap_jira/streams.py

```diff
@@ -407,13 +385,334 @@ class IssueStream(JiraStream):
     """
 
     name = "issues"
-    path = "/search?maxResults=1"
+    path = "/search?maxResults=10"
     primary_keys = ["id"]
-    replication_key = "updated"
+    replication_key = "id"
```

Comment:
```suggestion
    replication_key = "id"
```

Does this mean users will not get new records when an issue is updated?

Reply:
I have added a `get_url_params` function that fetches any record that is updated within the start/end data